pkcs11/_pkcs11.pyx

```diff
@@ -11,11 +11,14 @@ library loaded.
 from __future__ import (absolute_import, unicode_literals,
                         print_function, division)
 
-from posix cimport dlfcn
-
 from cython.view cimport array
 from cpython.mem cimport PyMem_Malloc, PyMem_Free
 
+IF UNAME_SYSNAME == "Windows":
+    from . cimport _mswin as mswin
+ELSE:
+    from posix cimport dlfcn
```

Comment:
Surprised no one has wrapped this actually.

Reply:
Not sure to get your point here (sorry, I'm not a native speaker). Could you clarify please?

Reply:
I'm surprised there's not a cross-platform library open for Cython.

Reply:
Oh OK. That would be great indeed!  I must admit I didn't look for it. If not existing, this is perhaps opportunity for a new project...